app/views/overrides/activation_keys/_host_environment_select.html.erb

```diff
@@ -34,18 +34,10 @@ end %>
   <input type="hidden" name="host[content_facet_attributes][id]" value="<%= @host.content_facet.try(:id) %>">
 <% end %>
 
-<% data_url = using_hostgroups_page? ? environment_selected_hostgroups_path : hostgroup_or_environment_selected_hosts_path %>
-<%= select_f f, :environment_id, Environment.all, :id, :to_label, {:include_blank => blank_or_inherit_f(f, :environment)},
-  {:label => _("Puppet Environment"), :onchange => 'update_puppetclasses(this);', 'data-url'=> data_url,
-   'data-content_puppet_match' => (@host || @hostgroup).new_record? || (@host || @hostgroup).content_and_puppet_match?,
-    :help_inline => %(<a id="reset_puppet_environment">#{_("Reset Puppet Environment to match selected Content View")}</a>).html_safe
-  } %>
-
-
```

Comment:
Also with this change it doesn't seem to be 'auto-selecting' the matching puppet environment for the selected content view and lifecycle environment.  Was this an intentional change?  I'm not necessarily opposed to it as the 'reset puppet environment' link still works, but was curious if intentional.